apps/scan/backend/scripts/copy_sheets.ts

```diff
@@ -87,13 +87,17 @@ function copySheets({ targetSheetCount }: CopySheetsInput): void {
     500 // A cap to limit how much data the script loads
   );
 
-  const newSheetIds = iter(store.forEachAcceptedSheet())
+  const sheets = iter(store.forEachAcceptedSheet())
     .take(maxNumSheetsToReadForCopying)
-    .cycle()
-    .take(numSheetsToCreate)
-    .map((sheet) => copySheet(store, sheet))
```

Comment:
Ah, because `copySheet` is accessing the db while the `forEachAcceptedSheet` iterator is still going.